boards/arm/bcm958402m2_a72/bcm958402m2_a72_defconfig

```diff
@@ -25,3 +25,9 @@ CONFIG_UART_CONSOLE=y
 CONFIG_PCIE_ENDPOINT=y
 CONFIG_PCIE_EP_IPROC=y
 CONFIG_PCIE_EP_IPROC_V2=y
+
+# Enable PAXDMA
+CONFIG_DMA=y
+CONFIG_DMA_64BIT=y
```

Comment:
What about DMA_PL330?

Reply:
Will enable in https://github.com/zephyrproject-rtos/zephyr/pull/32316